website/pages/docs/advanced-topics/monitoring/otel-datadog.md

```diff
@@ -1,66 +1,15 @@
 ---
-title: Monitoring CloudQuery
-description: Short walkthrough on how to use OpenTelemetry to monitor CloudQuery Syncs
+title: OpenTelemetry and Datadog
+description: Walkthrough on how to use OpenTelemetry to monitor CloudQuery Syncs with Datadog
 ---
 
-# Monitoring
```

Comment:
Everything from there until `OpenTelemetry and Datadog` (not including) was moved to a new `overview.md` file